arch/arm/core/aarch64/isr_wrapper.S

```diff
@@ -111,13 +111,6 @@ SECTION_FUNC(TEXT, _isr_wrapper)
 	/* Switch thread */
 	bl	z_arm64_context_switch
 
-	/* We return here in two cases:
```

Comment:
Now we will always return here right. There is no way out from ` z_arm64_context_switch`
And this seems now much simplified path.

Reply:
We will return here every time we swap in a task that was preemptively swapped out and yeah, the path is way more clear now.